CMakeLists.txt

```diff
@@ -311,7 +315,7 @@ include_directories(
     "${PROJECT_SOURCE_DIR}/src/contrib/nativefiledialog-extended/src/include"
 )
 
-if (WIN32)
+if (NOT RT64_BUILD_PLUGIN AND WIN32)
```

Comment:
Similar to what I said before, we can't be excluding D3D12 just because it's a plugin build.